content/docs/latest/installing/cloud/openstack.md

```diff
@@ -134,7 +134,7 @@ Boot the machines with the `openstack` CLI, referencing the image ID from the im
 
 ```shell
 openstack server create \
---user-data ./config.ign \
+--user-data ./ignition.json \
```

Comment:
Looks good to me, do you also want to fix it [here](https://github.com/flatcar/flatcar-website/blob/master/content/docs/latest/installing/cloud/openstack.md?plain=1#L195)